src/common/containers/GameArea.js

```diff
@@ -303,6 +317,8 @@ export class GameArea extends Component {
         onStartTutorial();
       } else if (this.urlMatchesGameMode('practice')) {
         this.tryToStartPracticeGame(match.params.deck);
+      } else if (location.pathname.startsWith('/sandbox')) {
```

Comment:
Should this endpoint string be specified in one place?

Reply:
I think this is fine for now.